lib/package.json

```diff
@@ -113,7 +113,8 @@
     "rollup": "^2.10.2",
     "rollup-plugin-commonjs": "^10.1.0",
     "rollup-plugin-node-resolve": "^5.2.0",
-    "rollup-plugin-size-snapshot": "^0.10.0",
+    "rollup-plugin-replace": "^2.2.0",
```

Comment:
commonjs, node-resolve, typescript and replace plugins are moved to `@rollup` scope.

Reply:
🙏